storage/innobase/include/page0page.h

```diff
@@ -57,6 +57,8 @@ this program; if not, write to the Free Software Foundation, Inc.,
                         ==============
 */
 
+extern bool srv_immediate_scrub_data_uncompressed;
```

Comment:
I kept it here, because it was declared here in mariadb/5.7. I can move it, but then I'll have to do that in 5.7 too - which will receive several more modificatiots, so I added this to the todo list too.